compiler/src/iree/compiler/Codegen/LLVMCPU/test/select_x86_64_lowering_strategy.mlir

```diff
@@ -1638,7 +1638,7 @@ func.func @pad_only() attributes {hal.executable.target = #executable_target_emb
 //  CHECK-SAME:     translation_info = #[[TRANSLATION]]
 //      CHECK:   tensor.pad {{.+}} {
 //      CHECK:     tensor.yield
-// CHECK-NEXT:   } {lowering_config = #[[CONFIG]]}
```

Comment:
or alternatively, only match `lowering_config = #[[COFIG]]` in this test to allow for other attributes